<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commande</class>
 <widget class="QDialog" name="commande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1437</width>
    <height>909</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commande</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 44, 52);
QDialog {background: transparent; }
QToolTip {
	color: #ffffff;
	background-color: rgba(27, 29, 35, 160);
	border: 1px solid rgb(40, 40, 40);
	border-radius: 2px;
}
*{
	
	font-size:18px;
};
border-radius: 5px;
gridline-color: transparent;
border-bottom: 1px solid rgb(44, 49, 60);
color: rgb(255, 96, 96);
QMainWindow {background: transparent; }
QToolTip {
	color: #ffffff;
	background-color: rgba(27, 29, 35, 160);
	border: 1px solid rgb(40, 40, 40);
	border-radius: 2px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 44, 52);/* LINE EDIT */
QLineEdit {
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}

/* SCROLL BARS */
QScrollBar:horizontal {
    border: none;
    background: rgb(52, 59, 72);
    height: 14px;
    margin: 0px 21px 0 21px;
	border-radius: 0px;
}
QScrollBar::handle:horizontal {
    background: rgb(85, 170, 255);
    min-width: 25px;
	border-radius: 7px
}
QScrollBar::add-line:horizontal {
    border: none;
    background: rgb(55, 63, 77);
    width: 20px;
	border-top-right-radius: 7px;
    border-bottom-right-radius: 7px;
    subcontrol-position: right;
    subcontrol-origin: margin;
}
QScrollBar::sub-line:horizontal {
    border: none;
    background: rgb(55, 63, 77);
    width: 20px;
	border-top-left-radius: 7px;
    border-bottom-left-radius: 7px;
    subcontrol-position: left;
    subcontrol-origin: margin;
}
QScrollBar::up-arrow:horizontal, QScrollBar::down-arrow:horizontal
{
     background: none;
}
QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
     background: none;
}
 QScrollBar:vertical {
	border: none;
    background: rgb(52, 59, 72);
    width: 14px;
    margin: 21px 0 21px 0;
	border-radius: 0px;
 }
 QScrollBar::handle:vertical {	
	background: rgb(85, 170, 255);
    min-height: 25px;
	border-radius: 7px
 }
 QScrollBar::add-line:vertical {
     border: none;
    background: rgb(55, 63, 77);
     height: 20px;
	border-bottom-left-radius: 7px;
    border-bottom-right-radius: 7px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }
 QScrollBar::sub-line:vertical {
	border: none;
    background: rgb(55, 63, 77);
     height: 20px;
	border-top-left-radius: 7px;
    border-top-right-radius: 7px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
 QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
     background: none;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }

/* CHECKBOX */
QCheckBox::indicator {
    border: 3px solid rgb(52, 59, 72);
	width: 15px;
	height: 15px;
	border-radius: 10px;
    background: rgb(44, 49, 60);
}
QCheckBox::indicator:hover {
    border: 3px solid rgb(58, 66, 81);
}
QCheckBox::indicator:checked {
    background: 3px solid rgb(52, 59, 72);
	border: 3px solid rgb(52, 59, 72);	
	background-image: url(:/16x16/icons/16x16/cil-check-alt.png);
}

/* RADIO BUTTON */
QRadioButton::indicator {
    border: 3px solid rgb(52, 59, 72);
	width: 15px;
	height: 15px;
	border-radius: 10px;
    background: rgb(44, 49, 60);
}
QRadioButton::indicator:hover {
    border: 3px solid rgb(58, 66, 81);
}
QRadioButton::indicator:checked {
    background: 3px solid rgb(94, 106, 130);
	border: 3px solid rgb(52, 59, 72);	
}

/* COMBOBOX */
QComboBox{
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding: 5px;
	padding-left: 10px;
}
QComboBox:hover{
	border: 2px solid rgb(64, 71, 88);
}
QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: top right;
	width: 25px; 
	border-left-width: 3px;
	border-left-color: rgba(39, 44, 54, 150);
	border-left-style: solid;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;	
	background-image: url(:/16x16/icons/16x16/cil-arrow-bottom.png);
	background-position: center;
	background-repeat: no-reperat;
 }
QComboBox QAbstractItemView {
	color: rgb(85, 170, 255);	
	background-color: rgb(27, 29, 35);
	padding: 10px;
	selection-background-color: rgb(39, 44, 54);
}

/* SLIDERS */
QSlider::groove:horizontal {
    border-radius: 9px;
    height: 18px;
	margin: 0px;
	background-color: rgb(52, 59, 72);
}
QSlider::groove:horizontal:hover {
	background-color: rgb(55, 62, 76);
}
QSlider::handle:horizontal {
    background-color: rgb(85, 170, 255);
    border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 9px;
}
QSlider::handle:horizontal:hover {
    background-color: rgb(105, 180, 255);
}
QSlider::handle:horizontal:pressed {
    background-color: rgb(65, 130, 195);
}

QSlider::groove:vertical {
    border-radius: 9px;
    width: 18px;
    margin: 0px;
	background-color: rgb(52, 59, 72);
}
QSlider::groove:vertical:hover {
	background-color: rgb(55, 62, 76);
}
QSlider::handle:vertical {
    background-color: rgb(85, 170, 255);
	border: none;
    height: 18px;
    width: 18px;
    margin: 0px;
	border-radius: 9px;
}
QSlider::handle:vertical:hover {
    background-color: rgb(105, 180, 255);
}
QSlider::handle:vertical:pressed {
    background-color: rgb(65, 130, 195);
}
QTableWidget{
	background-color: rgb(255, 255, 255);
}
QDialog {background: transparent; }
QToolTip {
	color: #ffffff;
	background-color: rgba(27, 29, 35, 160);
	border: 1px solid rgb(40, 40, 40);
	border-radius: 2px;
}
*{
	
	font-size:18px;
}

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="pushButton_43">
      <property name="geometry">
       <rect>
        <x>1230</x>
        <y>820</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	
	background-color: rgb(170, 0, 0);

}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/16x16/icons/16x16/cil-folder-open.png</normaloff>:/16x16/icons/16x16/cil-folder-open.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>20</y>
        <width>241</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Orbitron</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 0, 0);

border:2px solid  rgb(52, 59, 72);
font: 75 20pt &quot;Orbitron&quot;;
</string>
      </property>
      <property name="text">
       <string>COMMANDE</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>800</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QCheckBox::indicator:checked {
	background-color: rgb(170, 0, 0);

}
QCheckBox {
background-color: rgb(85, 170, 255);
color :rgb(255, 255, 255);
}
QCheckBox:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QCheckBox:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}
</string>
      </property>
      <property name="text">
       <string>Livraison</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QCheckBox" name="checkBox_4">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>790</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QCheckBox::indicator:checked {
	background-color: rgb(170, 0, 0);

}
QCheckBox {
background-color: rgb(85, 170, 255);
color :rgb(255, 255, 255);
}
QCheckBox:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QCheckBox:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}
</string>
      </property>
      <property name="text">
       <string>Sur place</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>620</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : rgb(85, 170, 255);

border:2px solid  rgb(52, 59, 72);
</string>
       </property>
       <property name="text">
        <string> ID client:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
color : rgb(255, 255, 255);
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>ID CLIENT</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>560</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : rgb(85, 170, 255);

border:2px solid  rgb(52, 59, 72);
</string>
       </property>
       <property name="text">
        <string> ID employé:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
color : rgb(255, 255, 255);
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>ID EMPLOYE</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>1070</x>
        <y>460</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 22pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>791</width>
        <height>391</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
	
	font: 87 10pt &quot;Cinzel Black&quot;;
    background-color: rgb(39, 44, 54);
    border-style: none;
    border-bottom: 1px solid rgb(85, 170, 255);
    border-right:  1px solid rgb(85, 170, 255);
}

QHeaderView::section:vertical
{
    border-left: 1px solid  rgb(85, 170, 255);
}
QTableView {
	
	color: rgb(255, 255, 255);
	border-style: solid;
	border-top: 1px solid  rgb(85, 170, 255);
	 border-left: 1px solid  rgb(85, 170, 255);
 border-bottom: 1px solid rgb(85, 170, 255);
 border-right:  1px solid rgb(85, 170, 255);
}
QTableView::item {
     border-bottom: 1px solid rgb(85, 170, 255);
    border-right:  1px solid rgb(85, 170, 255);
    background-color : none;
}
QTableView::indicator:unchecked {
    background-color: #d7d6d5
}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Orbitron</family>
        <pointsize>20</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 0, 0);

border:2px solid  rgb(52, 59, 72);
font: 75 20pt &quot;Orbitron&quot;;
</string>
      </property>
      <property name="text">
       <string>Menu</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_14">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>100</y>
        <width>591</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
	
	font: 87 10pt &quot;Cinzel Black&quot;;
    background-color: rgb(39, 44, 54);
    border-style: none;
    border-bottom: 1px solid rgb(85, 170, 255);
    border-right:  1px solid rgb(85, 170, 255);
}

QHeaderView::section:vertical
{
    border-left: 1px solid  rgb(85, 170, 255);
}
QTableView {
	
	color: rgb(255, 255, 255);
	border-style: solid;
	border-top: 1px solid  rgb(85, 170, 255);
	 border-left: 1px solid  rgb(85, 170, 255);
 border-bottom: 1px solid rgb(85, 170, 255);
 border-right:  1px solid rgb(85, 170, 255);
}
QTableView::item {
     border-bottom: 1px solid rgb(85, 170, 255);
    border-right:  1px solid rgb(85, 170, 255);
    background-color : none;
}
QTableView::indicator:unchecked {
    background-color: #d7d6d5
}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_33">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>800</y>
        <width>291</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	
	background-color:rgb(79, 158, 0);

}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
      </property>
      <property name="text">
       <string>ajouter un element dans la commande</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/16x16/icons/16x16/cil-folder-open.png</normaloff>:/16x16/icons/16x16/cil-folder-open.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_71">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>740</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font/>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color : rgb(85, 170, 255);

border:2px solid  rgb(52, 59, 72);
</string>
      </property>
      <property name="text">
       <string>ID </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_32">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>730</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
color : rgb(255, 255, 255);
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_32">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>800</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	
	background-color: rgb(170, 0, 0);

}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
      </property>
      <property name="text">
       <string>SUPP un element dans la commande</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/16x16/icons/16x16/cil-folder-open.png</normaloff>:/16x16/icons/16x16/cil-folder-open.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>610</y>
        <width>651</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Orbitron</family>
        <pointsize>12</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 0, 0);

border:2px solid  rgb(52, 59, 72);
font: 75 12pt &quot;Orbitron&quot;;
</string>
      </property>
      <property name="text">
       <string>Ajouter ou supprimer un element dans la COMMANDE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_34">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>10</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	border: 2px solid rgb(52, 59, 72);
	border-radius: 5px;	
	
	background-color: rgb(170, 0, 0);

}
QPushButton:hover {
	background-color: rgb(57, 65, 80);
	border: 2px solid rgb(61, 70, 86);
}
QPushButton:pressed {	
	background-color: rgb(35, 40, 49);
	border: 2px solid rgb(43, 50, 61);
}</string>
      </property>
      <property name="text">
       <string>Déconnexion</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/16x16/icons/16x16/cil-folder-open.png</normaloff>:/16x16/icons/16x16/cil-folder-open.png</iconset>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>520</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color : rgb(85, 170, 255);

border:2px solid  rgb(52, 59, 72);
</string>
       </property>
       <property name="text">
        <string>Id commande:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
color : rgb(255, 255, 255);
	background-color: rgb(27, 29, 35);
	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);
}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>ID commande</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>460</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 22pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>MONTANT :</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
